mmdet3d/datasets/transforms/loading.py

```diff
@@ -22,13 +23,38 @@ class LoadMultiViewImageFromFiles(object):
             Defaults to False.
         color_type (str, optional): Color type of the file.
             Defaults to 'unchanged'.
+        file_client_args (dict, optional): Config dict of file clients,
```

Comment:
remove `optional`, since we only use `optional` for the parameter whose default value is `None`.

Reply:
remove this